src/components/organisms/raw-json/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import ReactJson from "react-json-view"
+
+import BodyCard from "../body-card"
+
+type RawJSONProps = {
+  /**
+   * JSON data to render
+   */
+  data?: object
+  /**
+   * Body card title.
+   */
+  title: string
+}
+
+/**
+ * Renders a (collapsed) JSON tree section.
+ *
+ * @param {Object} props - React props
+ * @return {Object} - React element
+ */
+function RawJSON(props: RawJSONProps) {
+  const { title, data } = props
+
+  if (!data) {
+    return null
+  }
+
+  return (
+    <BodyCard className={"w-full mb-4 min-h-0 h-auto"} title={title}>
+      <div className="flex flex-col min-h-[100px] mt-4 bg-grey-5 px-3 py-2 h-full rounded-rounded">
+        <span className="inter-base-semibold">
+          Data <span className="text-grey-50 inter-base-regular">(1 item)</span>
+        </span>
+        <div className="flex flex-grow items-center mt-4">
+          <ReactJson name={false} collapsed={true} src={data} />
+        </div>
+      </div>
+    </BodyCard>
```

Comment:
I implemented `RawJSON` as a composition of `BodyCard` + `ReactJson` components since this exact section is used in 5 different places so it makes sense to me to extract it as an organism. This way the entire UI of the section is "encapsulated" in a single component. If we remove BodyCard, we are left with (more or less) `ReactJson` component which we could use directly and add the subtitle in the place of use.

Also, if the visual effect of a card is an issue, it could be controlled through a prop to achieve greater reusability?
I feel like this is one of those exceptions to the rule where I would prefer configuration through props over composition. But this is a matter of personal preference and I'm probably overthinking -- we could delete the component and use little repetition instead of making a wrong abstraction 🤷‍♂️ .

Reply:
Thanks for the explanation. You make a good point @fPolic 👍  Let's keep it as is for now! I don't think we will have the need for a generic raw JSON component in the foreseeable future, so let's pick up the discussion again if it becomes relevant again.